Formula/kontena-cli.rb

```diff
@@ -0,0 +1,185 @@
+class KontenaCli < Formula
+  desc "Command-line client for Kontena container & microservices platform"
+  homepage "https://kontena.io/"
+  url "https://github.com/kontena/kontena.git",
+      :tag => "v1.3.4",
+      :revision => "3588a33dfc992a0fb29603002ac33bfbe047c673",
+      :branch => "v1.3"
```

Comment:
branch isn't needed

Reply:
Without it a completely wrong version (1.1.x) gets installed for some reason.

Reply:
It's not possible for it to proceed unless the tag and revision match. You can try `:shallow => false` instead though if there's a problem.